### ISSUE TITLE
Invalid Latex generated. Causes ! Double subscript. in TexLive 

### ISSUE THREAD

**Issue by nasser1** [sagemath/sage#36197]
### Steps To Reproduce

These are the steps to reproduce. Call this integral using Maxima. It returns unevaluated.  Then calling latex on the result generates Latex which does not compile

```
>sage
┌────────────────────────────────────────────────────────────────────┐
│ SageMath version 10.1, Release Date: 2023-08-20                    │
│ Using Python 3.11.3. Type "help()" for help.                       │
└────────────────────────────────────────────────────────────────────┘
sage: var('_C0 _C1 _C2 _C3 _C4 _C5 _C6 _C7 _C8 _C9 x')
sage: anti=integrate((_C3*x^2-_C4)*(3*_C3*x^2+3*_C4+x)/x/((_C3*x^2+_C0*x+_C4)/(_C3*x^2+_C1*x+_C4))^(1/2)/(_C3^2*x^4+2*_C3*_C4*x^2+_C4^2-x^2),x,algorithm="maxima")

sage: anti
integrate((3*_C3*x^2 + 3*_C4 + x)*(_C3*x^2 - _C4)/((_C3^2*x^4 + 2*_C3*_C4*x^2 + _C4^2 - x^2)*x*sqrt((_C3*x^2 + _C0*x + _C4)/(_C3*x^2 + _C1*x + _C4))), x)

sage: latex(anti)
\int \frac{{\left(3 \, _{C_{3}} x^{2} + 3 \, _{C_{4}} + x\right)} {\left(_{C_{3}} x^{2} - _{C_{4}}\right)}}{{\left(_{C_{3}}^{2} x^{4} + 2 \, _{C_{3}} _{C_{4}} x^{2} + _{C_{4}}^{2} - x^{2}\right)} x \sqrt{\frac{_{C_{3}} x^{2} + _{C_{0}} x + _{C_{4}}}{_{C_{3}} x^{2} + _{C_{1}} x + _{C_{4}}}}}\,{d x}
sage: 

```

Copying the above to a Latex file and compiling gives

```
>lualatex main.tex
This is LuaHBTeX, Version 1.15.1 (TeX Live 2023/dev) 
 restricted system commands enabled.
(./main.tex
LaTeX2e <2022-11-01> patch level 1
 L3 programming layer <2023-01-24>
(/usr/local/texlive/2022/texmf-dist/tex/latex/base/article.cls
Document Class: article 2022/07/02 v1.4n Standard LaTeX document class
(/usr/local/texlive/2022/texmf-dist/tex/latex/base/size12.clo))
(/usr/local/texlive/2022/texmf-dist/tex/latex/amsmath/amsmath.sty
For additional information on amsmath, use the `?' option.
(/usr/local/texlive/2022/texmf-dist/tex/latex/amsmath/amstext.sty
(/usr/local/texlive/2022/texmf-dist/tex/latex/amsmath/amsgen.sty))
(/usr/local/texlive/2022/texmf-dist/tex/latex/amsmath/amsbsy.sty)
(/usr/local/texlive/2022/texmf-dist/tex/latex/amsmath/amsopn.sty))
(/usr/local/texlive/2022/texmf-dist/tex/latex/l3backend/l3backend-luatex.def)
(./main.aux) (/usr/local/texlive/2022/texmf-dist/tex/latex/base/ts1cmr.fd)
! Double subscript.
<argument> {\left (_{C_{3}}^{2} x^{4} + 2 \, _{C_{3}} _
                                            {C_{4}} x^{2} + _{C_{4}}^{2} - x^{2
l.53 \end{align*}
               
? 


```

This is the latex file with the latex generated by sagemath

```
\documentclass[12pt,titlepage]{article}
\usepackage{amsmath}
\begin{document} 
\begin{align*}
\int \frac{{\left(3 \, _{C_{3}} x^{2} + 3 \, _{C_{4}} + x\right)} {\left(_{C_{3}} x^{2} - _{C_{4}}\right)}}{{\left(_{C_{3}}^{2} x^{4} + 2 \, _{C_{3}} _{C_{4}} x^{2} + _{C_{4}}^{2} - x^{2}\right)} x \sqrt{\frac{_{C_{3}} x^{2} + _{C_{0}} x + _{C_{4}}}{_{C_{3}} x^{2} + _{C_{1}} x + _{C_{4}}}}}\,{d x}
\end{align*}
\end{document}


```

### Expected Behavior

Expected that valid latex to be generated

### Actual Behavior

Invalid latex is generated from the latex() command

### Additional Information

Here is a link describing this latex error

[https://www.overleaf.com/learn/latex/Errors/Double_subscript](https://www.overleaf.com/learn/latex/Errors/Double_subscript)

### Environment

```markdown
- **OS**:
Linux Manjaro
- **Sage Version**:
10.1
```

### Checklist

- [X] I have searched the existing issues for a bug report that matches the one I want to file, without success.
- [X] I have read the documentation and troubleshoot guide

**Comment by fchapoton** [comment]
well, avoid using variables names starting with underscore.

**Comment by DaveWitteMorris** [comment]
I think this can be closed as a duplicate of #32422.

**Comment by fchapoton** [comment]
yes,let's close
